src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate: function(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
+    reset: function() {
+      this.result = 0;
+
+      return this;
+    },
+    add: function(result, number) {
```

Comment:
```suggestion
    add: function(number) {
```
these methods should not receive result as a prop, use this.result inside